src/output/icons.rs

```diff
@@ -429,16 +456,16 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "guardfile"      => Icons::LANG_RUBY,        // 
     "gv"             => '\u{f1049}',             // 󱁉
     "gz"             => Icons::COMPRESSED,       // 
-    "h"              => Icons::LANG_C,           // 
+    "h"              => Icons::HEADER,           // 
```

Comment:
We just changed these to language specific files.  We would not want to revert these back to header.

Reply:
Oh sorry. It does not have to have its own constant, but I like the "H" icon. I will change if others don't want.